core/templates/pages/volunteer-page/volunteer-page.component.html

```diff
@@ -193,119 +270,174 @@ <h3 class="oppia-volunteer-h3">{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION' | tran
               </ngb-carousel>
 
             </div>
-            <div class="col oppia-volunteer-tab-description">
-              <p>{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON_CREATION_CONTENT_1' | translate }}</p>
-              <p>{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON_CREATION_CONTENT_2' | translate }}</p>
-
-              <div class="oppia-volunteer-dotted-spaced-line mb-5"><br></div>
-
-              <p [innerHTML]="'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON_CREATION_WHAT_WE' | translate "></p>
-              <ul>
-                <li class="oppia-volunteer-page-description-list-item">{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON_CREATION_WHAT_WE_1' | translate }}</li>
-                <li class="oppia-volunteer-page-description-list-item">{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON_CREATION_WHAT_WE_2' | translate }}</li>
-                <li class="oppia-volunteer-page-description-list-item">{{ 'I18N_VOLUNTEER_PAGE_VOLUNTEER_SECTION_LESSON_CREATION_WHAT_WE_3' | translate }}</li>
-              </ul>
-
-            </div>
           </div>
         </mat-tab>
       </mat-tab-group>
     </div>
   </div>
 
-  <div class="oppia-volunteer-footer">
-    <p [innerHTML]="'I18N_VOLUNTEER_PAGE_FOOTER' | translate"></p>
+  <div class="oppia-volunteer-footer"
+       [style.background-image]="'url(' + getStaticImageUrl(getWebpExtendedName(footerImgPath)) + '), url(' + getStaticImageUrl(footerImgPath) + ')'"
+       [style.background-position]="'center center'"
+       [style.background-size]="'cover'">
+    <p>{{ 'I18N_VOLUNTEER_PAGE_FOOTER' | translate }}</p>
 
     <button class="btn">
       <a href="https://forms.gle/rhFYoLLSFr3JEZHy8"
          target="_blank"
-         rel="noopener"
-         [innerHTML]="'I18N_VOLUNTEER_PAGE_FOOTER_BUTTON' | translate">
+         rel="noopener">
+         {{ 'I18N_VOLUNTEER_PAGE_FOOTER_BUTTON' | translate }}
       </a>
     </button>
   </div>
 </div>
 
 <style>
+  .volunteer-page button {
+    background-color: #00645c;
+    border-radius: 4px;
+    height: 40px;
+    margin: 2rem auto 7rem auto;
+  }
+
+  .volunteer-page button:hover,
+  .volunteer-page button:focus {
+    background-color: #3d9991;
+  }
+
+  .volunteer-page button a {
+    color: rgba(255, 255, 255, 1);
```

Comment:
```suggestion
    color: rgba(255, 255, 255);
```
replace with rgb as alpha channel is unused here (full opacity). Also should we use hex color codes as we are doing elsewhere? (#fff)

Reply:
done : )